internal/usage/usage.go

```diff
@@ -38,9 +38,9 @@ const (
 	AtlasAuthDB              = "Authentication database name. If the user authenticates with AWS IAM, x.509, or LDAP, this value should be $external. If the user authenticates with SCRAM-SHA, this value should be admin."
 	Granularity              = "ISO 8601-formatted duration that specifies the interval between measurement data points. Only the following subset of ISO 8601-formatted time periods are supported: PT10S, PT1M, PT5M, PT1H, P1D. When you specify granularity, you must specify either period or start and end."
 	Page                     = "Page number that specifies a page of results."
-	Forever                  = "Acknowledges an alert 'forever'."
-	Status                   = "Alert's status. The values are TRACKING, OPEN, CLOSED, and CANCELED."
-	Until                    = "Time, in ISO 8601 format, until which the alert has been acknowledged. mongocli returns this value after the alert has been acknowledged."
+	Forever                  = "Acknowledges an alert 'forever'. You can't set both forever and until in the same command."
+	Status                   = "State of this alert. Valid values are TRACKING, OPEN, CLOSED, and CANCELLED."
```

Comment:
you may need to add `//nolint:gospell` here

Reply:
@gssbzn Thank you, that didn't work but I found a solution that seems to work from this PR on the golangci repo: https://github.com/golangci/golangci-lint/commit/353cd2c28b17379fd0e3f0e91cb0180a24bd3392. RFAL when you have time